README.md

```diff
@@ -6,14 +6,18 @@ This repo houses the assets used to build the website for Longhorn, available at
 
 ## Running the site locally
 
-To run the website locally, you need to have the [Hugo](https://gohugo.io) static site generator installed (installation instructions [here](https://gohugo.io/getting-started/installing/)). Once Hugo is installed:
+To run the website locally, you need to have the [Hugo](https://gohugo.io) static site generator installed (installation instructions [here](https://gohugo.io/getting-started/installing/)). You can find the latest release and pre-compiled binaries on the [Hugo releases page](https://github.com/gohugoio/hugo/releases). 
+
+**Important:** This project requires the "extended" edition of Hugo to process SCSS/SASS files.
+
+Once Hugo is installed:
 
 ```bash
 yarn install
 hugo server --buildDrafts --buildFuture
 ```
 
-This starts Hugo in local mode. You can see access the site at http://localhost:1313.
+This starts Hugo in local mode. You can see access the site at http://localhost:8085.
```

Comment:
Is this valid? After running the commands, it looks like port 1313 is being used.
```
Web Server is available at http://localhost:1313/ (bind address 127.0.0.1) 
Press Ctrl+C to stop
```

Reply:
You are right.
1313 is for `hugo server --buildDrafts --buildFuture`
8085 works for [makefile](https://github.com/longhorn/website/blob/master/Makefile#L7)